_CS141/part2.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: CS141
+part: true
+title: Lazy Evaluation & Recursion
+---
+
+## Evaluation Strategies
+
+> **Evaluation strategies** determine the order in which we reduce expressions. An expression that can be reduced is called a **redex.**
+
+### Strictness
+
+> A programming language is **strict** if the arguments of a function are evaluated before the function is called.
```

Comment:
This is more a description of call-by-value, which happens to be a strict evaluation strategy